src/transformers/modeling_tf_albert.py

```diff
@@ -826,3 +826,68 @@ def call(self, inputs, **kwargs):
         outputs = (logits,) + outputs[2:]  # add hidden states and attention if they are here
 
         return outputs  # logits, (hidden_states), (attentions)
+
+
+@add_start_docstrings(
+    """Albert Model with a span classification head on top for extractive question-answering tasks like SQuAD (a linear layers on top of the hidden-states output to compute `span start logits` and `span end logits`). """,
+    ALBERT_START_DOCSTRING,
+)
+class TFAlbertForQuestionAnswering(TFAlbertPreTrainedModel):
+    def __init__(self, config, *inputs, **kwargs):
+        super(TFAlbertForQuestionAnswering, self).__init__(config, *inputs, **kwargs)
```

Comment:
unneeded Python 2 remnant, it's now just `super()`

Reply:
Oh ok thank you I'm gonna change that, I was a bit lost by the different ways of doing XD